library/odoo

```diff
@@ -0,0 +1,4 @@
+# maintainer: Aaron Bohy <aab@odoo.com> (@aab-odoo)
+
+8.0: git://github.com/odoo/docker@2aae53084d32aaf44d3630e2a09d685bf971d81c 8.0
+8: git://github.com/odoo/docker@2aae53084d32aaf44d3630e2a09d685bf971d81c 8.0
```

Comment:
"latest" is actually a misnomer, unfortunately.  "latest" doesn't actually
mean "latest".  It really should've been "default" or something more to
that effect.  That's just the tag that will be used when someone does
"docker run odoo" without a tag -- it'll default to running "odoo:latest",
which should be the default recommended version.  For example, in the
Ubuntu image, "latest" is set to the latest LTS, not the latest release
(14.04 vs 14.10).